src/image_bios.cpp

```diff
@@ -198,15 +202,59 @@ void BIOSUpdater::unlock()
 
 void BIOSUpdater::doInstall(const fs::path& file)
 {
-    constexpr auto mtd = "/dev/mtd/bios";
-
     // NOTE: This process may take a lot of time and we want to show the
     //       progress from original flashcp output.
-    printf("Writing %s to %s\n", file.filename().c_str(), mtd);
-    int rc = system(strfmt("flashcp -v %s %s", file.c_str(), mtd).c_str());
+    printf("Writing %s to %s\n", file.filename().c_str(), mtdDevice);
+    int rc =
+        system(strfmt("flashcp -v %s %s", file.c_str(), mtdDevice).c_str());
     checkWaitStatus(rc, "");
 }
 
+void BIOSUpdater::doBeforeInstall(bool reset)
+{
+    if (!reset)
+    {
+        puts("Preserving BIOS settings...");
```

Comment:
I guess we need to call it `UEFI` everywhere. It's not a BIOS. Also, this preserves not just the UEFI settings, but also the settings for x722 adapters, doesn't it? Maybe say "Preserving NVRAM" ? What do you think?

Reply:
I think "UEFI" will be more familiar for an end user.